.github/workflows/rust.yml

```
@@ -2,132 +2,103 @@ name: Rust CI
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly
-          override: true
-      - uses: Swatinem/rust-cache@v2
-      - name: Generate documentation
-        run: cargo doc --lib --no-deps --all-features --document-private-items
-        env:
-          RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg docsrs
-          RUSTDOCFLAGS: --cfg docsrs -Dwarnings
+    - uses: actions/checkout@v4
+    - uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)